src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs

```diff
@@ -195,8 +203,27 @@ impl Sandbox for VirtSandbox {
         self.execute_oci_hook_functions(&prestart_hooks, &create_runtime_hooks, state)
             .await?;
 
-        // TODO: if prestart_hooks is not empty, rescan the network endpoints(rely on hotplug endpoints).
-        // see: https://github.com/kata-containers/kata-containers/issues/6378
+        let has_prestart_hooks = !prestart_hooks.is_empty() || !create_runtime_hooks.is_empty();
```

Comment:
Suggest wrapping into the function `process_after_start_vm`.

Reply:
fixed